queries/idris/highlights.scm

```diff
@@ -0,0 +1,200 @@
+; ------------------------------------------------------------------------------
+; Literals and comments
+(integer) @number
+
+(literal
+  (number)) @number.float
+
+(char) @character
+
+[
+  (string)
+  (pat_string)
+  (triple_quote_string)
+] @string
+
+(comment) @comment
+
+; ------------------------------------------------------------------------------
+; Punctuation
+[
+  "("
+  ")"
+  "{"
+  "@{"
+  "}"
+  "["
+  "[<"
+  "]"
+] @punctuation.bracket
+
+(comma) @punctuation.delimiter
```

Comment:
Also `(colon)` and `(pat_op)` and `(tuple_operator)` (unrelated but these should probably only be anonymous nodes)

Reply:
can do it, but why should by anonymous? (teach me, please)

Reply:
anonymous == not named (e.g.,  `"="` vs. `(equal_sign)`).

Reply:
it was hard, but I understood

Reply:
@ribru17 friendly ping

Reply:
Oops sorry!

> do You mean I should add them as
> 
> ```
> [
>  (comma)
>  (colon)
>  (pat_op)
>  (tuple_operator)
> ] @punctuation.delimiter
> ```
> 
> ???

Yes, correct. You can ignore my comment about the anonymous nodes. I only meant that in the *future*, it would be nice for the parser to expose them as, e.g, `","` rather than `(comma)`. But don't worry about it if it overcomplicates things.